packages/commonwealth/client/scripts/views/components/component_kit/CWSidebarHeader/CWSidebarHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import 'components/component_kit/CWSidebarHeader/CWSidebarHeader.scss';
+import React from 'react';
+
+import app from 'state';
+import { CWCommunityAvatar } from '../../component_kit/cw_community_avatar';
+import { navigateToCommunity, useCommonNavigate } from 'navigation/helpers';
+
+const SidebarHeader = () => {
+  const navigate = useCommonNavigate();
+
+  return (
+    <div className="SidebarHeader">
+      <CWCommunityAvatar
+        community={app.chain && app.chain.meta}
+        onClick={() =>
+          navigateToCommunity({ navigate, path: '', chain: app.chain.id })
+        }
+      />
+      {app.chain && <p className="header">{app.chain.meta.name}</p>}
```

Comment:
use `CWText` with correct prop instead of `p`. Right know it is 16px but according to the designs it should be 18px. In figma in right sidebar, you can find description what kind of CWText type it is. In this case, Heading5 which is `h5` type.

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/fe7cc758-7931-4ba4-b2d5-e5bedaa1a722)